app/src/sharedTest/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -229,7 +229,7 @@ class TopicPracticeFragmentTest {
   @Test
   // TODO(#973): Fix TopicPracticeFragmentTest
   @Ignore
-  fun testTopicPracticeFragment_loadFragment_selectSubtopics_deselectsubtopics_startButtonIsInactive() { // ktlint-disable max-line-length
+  fun testTopicPracticeFragment_selectSubtopics_deselectsubtopics_startButtonIsInactive() {
```

Comment:
``deselectSubtopics`` (there are two words here, so they should be made distinct via capitalization).